clouds/clouds-azure/src/test/resources/azure/armTemplates/eap.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "type": "string",
+      "defaultValue": "sunstoneVM"
+    },
+    "imageRefId": {
+      "type": "string",
+      "defaultValue": "/subscriptions/7dee6f21-9f05-414e-99fa-08d3215fb420/resourceGroups/istraka-test/providers/Microsoft.Compute/images/eap-test-image"
```

Comment:
replace with `AzureTestConstants.IMAGE_REF` ?

Reply:
I removed the default value. It still comes from test as a parameter.